packages/@react-aria/button/src/useButton.ts

```diff
@@ -67,7 +67,7 @@ export function useButton(props: AriaButtonProps, ref: RefObject<HTMLElement>):
   });
 
   let {focusableProps} = useFocusable(props, ref);
-  let buttonProps = mergeProps(pressProps, focusableProps);
+  let buttonProps = mergeProps(focusableProps, pressProps);
```

Comment:
order mattered? why?

Reply:
a `tabIndex` might come through the press context, and that should override the default one set by `useFocusable`.